### ISSUE TITLE
On OS X Pushing "Start" returns "Invalid dataserv-client" error

### ISSUE THREAD

**Issue by JeremyPepper2** [Storj/driveshare-gui#120]
I am runningg version 2.1.4 of the dataserv-client software.  After installing the re-released copy of the DriveShare GUI, clicking start produces an error message saying I have an "Invalid dataserv-client" installed.

I run my dataserv-client off of Python 3 and didn't have any luck playing around with trying to use Python 2 so I could upgrade.  I think DriveShare should take the latest version capable of running on both Python 2 and Python 3 - at least until both versions are capable of running the client.

I'm assuming my assessment is correct and this is a bug/oversight.

**Comment by littleskunk** [comment]
I bet you are running driveshare-gui 0.4.1 and not 0.4.2. Please open the gui menu Help->Abount and check the version.

btw: driveshare-gui is using the dataserv-client osx binary. You dont need to install any python version to run it. You can install dataserv-client yourself but the gui will ignore the installation.

**Comment by JeremyPepper2** [comment]
I'm running 0.4.2

**Comment by littleskunk** [comment]
Please delete /Library/Application Support/DriveShare/dataserv-client.app and restart the gui.

**Comment by JeremyPepper2** [comment]
That worked.  THANKS!

Also, just navigating to the directory path you gave did not work (/Users/<user_name>/Library/Application Support.  For some reason the "DriveShare" subfolder (and a BUNCH of others) only showed up when I copied the file extension you gave above into Finder's "Go to folder" feature.  I.e., I had to type the following path into "Go to folder" in order to get the "DriveShare" subfolder to show up:

/Users/<user_name>/Library/Application Support

Simply navigating through Finder to that same path only displayed a handful of files... frustrating!
